recipes/psyplot-gui/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "psyplot-gui" %}
+{% set version = "1.0.0.post1" %}
+{% set sha256 = "b238f93b911b6f760c19394ed27304066dd6d42f01160e5fe0c61a5192212397" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - psyplot
```

Comment:
There will be a race condition when we merge this and packages that depended on packages that are not yet build will fail. It is up to you if you want to submit separate PRs or if you want to manually find the failures and restart the CIs later.